Add troubleshooting information for fisher module

KattTails opened the pull request:
Hi, as per #733 I've updated the readme.md with some troubleshooting for the fisher overlay. I'll try and keep the troubleshooting doc updated with more common problems if they arise.

quisquous commented on README.md:
```diff
@@ -237,9 +237,13 @@ When you cast your line at a fishing hole, this module keeps track of when you r
 
+You will need to ensure the chat filter for `[ ] Own Gatherer Messages` is enabled in at least one of your chat tabs and that the option `[ ] Hide Chat Log (for privacy)` is not enabled in ACT FFXIV Plugin's options. 
+
+See [here](https://gist.github.com/KattTails/a67527be56be64e22b9f2eb962d5cf4f#file-cactbot-fisher-troubleshooting-md) for instructions for this and other common problems.
```
Do you want to include this inside the cactbot repository proper? Maybe just by adding a docs/readme-fisher.md or docs/fisher-troubleshooting-md file?

KattTails replied:
You're right it would be more suited in such a file, not the main `readme.md`. I was just concerned people wouldn't find it/think to look there and you'd end up with the same number of issues created about it. 

The docs/ folder only really has 'advanced' documentation in it currently, perhaps it could be useful to have a general-troubleshooting.md with common cactbot issues similar to [ngld's FAQ](https://gist.github.com/ngld/e2217563bbbe1750c0917217f136687d#my-act-isnt-showing-any-numbers-what-can-i-do). You could then move the `Potential errors and workarounds` section from the readme.md too and link there instead.

quisquous replied:
I think this documentation would fit in the docs/ folder, and I'm not sure I understand how linking to a gist is more findable than linking to a markdown file in the repository.  I definitely agree that there could also be a potential errors and workarounds doc too!

KattTails replied:
Ah I meant on the rare occasions people go to check the readme before asking for help ^^
I thought if someone was having an issue with the fisher module for example, they'd go look in the fisher module section on the main page, not find anything and give up. I'll look into building a list of common problems & solutions and drafting up some F.A.Q style document.

quisquous replied:
Sorry if I wasn't clear--I was trying to propose adding a doc in the docs/ folder and then linking it from both the main documentation talking about the fisher module as well as an FAQ.

quisquous replied:
I think really my objection here is to have half the documentation about fishing in the cactbot repo and half in a gist.  I think it would make more sense to have it all in the cactbot repo, and instead of linking to a gist, instead link to some markdown file in docs/ so it can all be centrally located, if that makes sense?

I think the other changes to the main README.md are fine.

KattTails commented:
Ahh sorry for forgetting about this! I've added an FAQ into 'docs\` and added a link on the main `readme.md` to it. I've moved the `Common workarounds and solutions` section into the new FAQ and updated it for ngld's OverlayPlugin. I included a few of the common errors I've seen on the FFXIV ACT Discord too, and added the fishing module info underneath. Your thoughts on these changes?

quisquous commented on README.md:
```diff
@@ -215,6 +204,8 @@ Cast times are currently only logged as you fish, so there won't be any data unt
 
 [See here](https://www.youtube.com/watch?v=GHgWIA-Zhug) for examples of the different tug types.
 
+Check [here](docs/Fisher-Troubleshooting.md) for common troubleshooting tips.
```
I assume you mean docs/FAQ-TroubleShooting.md#fisher-module?

KattTails replied:
Oops. Yes, that would be correct. Originally it was two files but it felt unnecessary. I've changed it.
